samples/logging-sample/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/GoogleCloudPlatform/cloud-build-notifiers/samples/logging-sample
+
+go 1.15
+
+require (
+	github.com/GoogleCloudPlatform/cloud-build-notifiers/lib/notifiers v0.0.0-20210205212514-9176fa6ca224 // indirect
```

Comment:
The problem was that using the local import required that I run the commands from the root directory.  Since the README is in the samples directory, it would require a lot of directions to `cd` up, which I wanted to avoid.  Also it changes the commands in the `cloudbuild` and `Dockerfile` to look for subdirectories.  As a sample, I think it's better to stand alone.  WDYT?